terraform/amazon/modules/network-existing-vpc/route53.tf

```diff
@@ -5,7 +5,7 @@ resource "aws_route53_zone" "private" {
   comment       = "Hosted zone for private kubernetes in ${var.environment}"
 
   vpc {
-    vpc_id = "${aws_vpc.main.0.id}"
```

Comment:
Why did we not see this error?

Reply:
I guess we didn't use an existing VPC since we updated terraform and so used the new variable assignments. Something to look out for